test/650_proxy_env_test.sh

```diff
@@ -2,6 +2,18 @@
 
 . ./config.sh
 
+CMD="run -e WEAVE_CIDR=10.2.1.4/24 $SMALL_IMAGE $CHECK_ETHWE_UP"
+
+check() {
+  assert_raises "eval '$(weave_on $HOST1 env)' ; docker $CMD"
+  assert_raises "docker $(weave_on $HOST1 config) $CMD"
+
+  # Check we can use the weave script through the proxy
+  assert_raises "eval '$(weave_on $HOST1 env)' ; $WEAVE version"
+  assert_raises "eval '$(weave_on $HOST1 env)' ; $WEAVE ps"
+  assert_raises "eval '$(weave_on $HOST1 env)' ; $WEAVE launch-router"
```

Comment:
I know this was there before, but now we invoke this twice.... wouldn't it suffice to have *one* of these three tests, e.g. just the version test?